src/libraries/System.Collections/src/System/Collections/Generic/PriorityQueue.cs

```diff
@@ -40,19 +40,9 @@ public class PriorityQueue<TElement, TPriority>
         /// </summary>
         private int _version;
 
-        /// <summary>
-        /// When the underlying buffer for the heap nodes grows to accomodate more nodes,
-        /// this is the minimum the capacity will grow by.
-        /// </summary>
-        private const int MinimumElementsToGrowBy = 4;
-
-        /// <summary>
-        /// The index at which the heap root is maintained.
-        /// </summary>
-        private const int RootIndex = 0;
-
         /// <summary>
         /// Specifies the arity of the d-ary heap, which here is quaternary.
+        /// It is assumed that this value is a power of 2.
```

Comment:
You could add:
```C#
#if DEBUG
    static PriorityQueue()
    {
        Debug.Assert(...); // whatever you want to validate
    }
#endif
```
if there are any such things you think it's important to validate.

Reply:
Addressed